packages/x-grid-data-generator/src/renderer/renderTotalPrice.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import clsx from 'clsx';
-import { makeStyles, createStyles, Theme, fade } from '@material-ui/core/styles';
-import { CellParams } from '@material-ui/x-grid';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+import { CellParams, muiStyleAlpha } from '@material-ui/x-grid';
```

Comment:
We can't import modules from x-grid. It breaks the codesandbox demos that depend on @material-ui/data-grid: https://codesandbox.io/s/9gbe4.

Reply:
I can think of two options:

1. We make the dependency explicit, we use data-grid instead of x-grid as they are equivalent and is more friendly (MIT). We fix the codesandbox dependency list generation to include the missing dependency
2. We remove the peer dependency on x-grid https://unpkg.com/@material-ui/x-grid-data-generator@4.0.0-alpha.17/package.json.

Reply:
We can continue the discussion here https://github.com/mui-org/material-ui-x/pull/887.

I've opened a PR with the potential change.